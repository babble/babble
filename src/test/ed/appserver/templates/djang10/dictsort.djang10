{% for num in array2|dictsort:"myProp" %}
    {{ num.myProp }}
{% endfor %}

{% for num in array2|dictsortreversed:"myProp" %}
    {{ num.myProp }}
{% endfor %}