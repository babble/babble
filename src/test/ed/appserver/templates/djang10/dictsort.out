12345678910
10987654321